src/Tactic/Cong.agda

```diff
@@ -55,6 +55,9 @@ open import Reflection.AST.Term                 as Term
 
 open import Reflection.TCM.Syntax
 
+⌞_⌟ : ∀ {a} {A : Set a} → A → A
+⌞_⌟ x = x
```

Comment:
Moving a little bit of the discussion you've written here, documenting the purpose of this would be great.

Reply:
Yes! Agreed and done.